src/components/departure-date-selector/departure-date-selector.module.css

```diff
@@ -0,0 +1,113 @@
+.departureDateSelector {
+  display: flex;
+  flex-direction: column;
+  /* TODO: Spacing not in variables. Use variable instead? */
```

Comment:
The design should be revised to follow the correct spacing I think

Reply:
Agree, I'd prefer that over adding new values. Though a bit strange (maybe not the best word) that a spacing of 1rem is not in the set of spacings? Same with 0.75rem for border-radius.

Should we just keep this values for now until we have a design review, or similar to update the spacings, or use the spacings we have for now?

Reply:
Don't think 16px is ever used in spacing in the webshop or app 🤔

Reply:
Probably not since it is not a spacing.

The ones we are using at the moment are:
6px / 0.375rem (spacing between date/time inputs and type)
16px / 1rem (margin-bottom for search box asks for this, have used --spacings-medium)
12px / 0.75rem (**border-radius**)